shared/util/component-processor/src/jvmMain/kotlin/com/rudolfhladik/componentprocessor/ComponentProcessor.kt

```diff
@@ -0,0 +1,57 @@
+package com.rudolfhladik.componentprocessor
+
+import com.google.devtools.ksp.processing.CodeGenerator
+import com.google.devtools.ksp.processing.Dependencies
+import com.google.devtools.ksp.processing.KSPLogger
+import com.google.devtools.ksp.processing.Resolver
+import com.google.devtools.ksp.processing.SymbolProcessor
+import com.google.devtools.ksp.symbol.KSAnnotated
+import com.google.devtools.ksp.symbol.KSClassDeclaration
+import com.rudolfhladik.annotation.Component
+import com.rudolfhladik.componentprocessor.content.ComponentContentGenerator
+import java.io.OutputStream
+import kotlin.reflect.KClass
+
+class ComponentProcessor(
+    private val codeGenerator: CodeGenerator,
+    private val logger: KSPLogger,
```

Comment:
Logger is unused, wanna keep it in? (I'm literally doing Danger's work here, which reminds me there should be `ktlint` and `danger` plugins added in `component-processor` and `component-annotations` Gradle projects)